dali/operators/audio/mel_scale/mel_filter_bank.cc

```diff
@@ -23,8 +23,10 @@ DALI_SCHEMA(MelFilterBank)
     .DocStr(R"code(Converts a spectrogram to a mel spectrogram by applying a bank of
 triangular filters.
 
-Expects an input with at least 2 dimensions where the last two dimensions correspond to
-the fft bin index and the window index, respectively.
+Expects an input with at least 2 dimensions.
+
+Please note that the CPU implementation supports only the layout with the last two
```

Comment:
How hard would be to rework CPU implementation as well?

Reply:
It should use a separate implementation of the actual processing loop, but it should some 30-50 lines or so (if the kenrel isn't totally botched).

Reply:
I plan to implement the CPU part. I can do it by contributing to this PR so that we don't introduce the note about the lack of support in the CPU version

Reply:
It's not a problem in nightly, IMO. What I would not like to see is this partial support making it into tagged release.